lib/commands/dumpPlugin.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+const Chalk = require('chalk');
+const Program = require('commander');
+const ShopwareStoreCommander = require('../shopwareStoreCommander');
+const util = require('../util');
+const programVersion = require('../version');
+const spinner = require('../cliStatusIndicator');
+const getPassword = require('../passwordPrompt');
+
+// Define CLI
+Program
+    .version(programVersion)
+    .arguments('<file>')
+    .option('-u, --username <username>', 'The shopware username.')
+    .option('-p, --plugin <plugin>', 'The name of the plugin for which to output information')
+    .option('-t, --output-format <format>', 'Output in format <format>', 'json')
+    .parse(process.argv);
+
+
+// Load an .env config if available
+require('../envLoader').loadConfig(Program);
+
+async function main() {
+    try {
+        // Validate arguments
+        if (typeof Program.username === 'undefined') {
+            console.error(Chalk.white.bgRed.bold('No username given!'));
+            process.exit(1);
```

Comment:
We should just `throw` here, since the general `catch` block does the same output (plus the exception, which is fine).

Reply:
Ah sorry. I thought I checked other commands for comparison, but it turns out they don't `throw`. So to make it consistent, please move the validation out of the main execution function and just `exit()` when necessary.

Reply:
You probably just recognized that having everything in a main function is the standard way of doing this and assumed there's nothing wrong with it – because there isn't ;)

I did change it back, now.

But I really think this is such a nitpick that should hardly ever happen during a review – it really seems to me that type of change is nothing but a frustrating time waster out of which nothing is gained for anyone.